dask-gateway/dask_gateway/dask_cli.py

```diff
@@ -574,7 +574,7 @@ async def start_worker(
     nthreads=1,
     memory_limit="auto",
     scheduler_address=None,
-    local_directory="",
+    local_directory=None,
```

Comment:
Did distributed change how the `local_directory` parameter is handled?

Reply:
Looks like this was https://github.com/dask/distributed/pull/3441

Reply:
And for reference, https://github.com/dask/distributed/pull/3964 is achieving the same goal (fall back to config / cwd) but by changing the implementation rather than the default. Dunno what the long term plan is.